localstack/mock/infra.py

```diff
@@ -368,7 +368,7 @@ def update_apigateway(method, path, data, headers, response=None, return_forward
 
 def update_kinesis(method, path, data, headers, response=None, return_forward_info=False):
     if return_forward_info:
-        if KINESIS_RETURN_ERRORS:
+        if os.environ['KINESIS_RETURN_ERRORS'] == 'True':
```

Comment:
Instead of querying os.environ directly, could we have a config variable (in constants.py) that is initialized with os.environ? Something like
```
KINESIS_RETURN_ERRORS = os.environ.get('KINESIS_RETURN_ERRORS') == 'True'
```
Then we can just use that variable in the if statement:
```
if constants.KINESIS_RETURN_ERRORS:
    ...
```